### ISSUE TITLE
Can't build on fedora 33

### ISSUE THREAD

**Issue by Redcroft** [GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD#392]
I get the following error when trying to compile on Fedora33 (using HFS18.5.351)

```
...
[ 68%] Built target hdRpr_headerfiles
[ 69%] Generating files
Traceback (most recent call last):
  File "/home/redsith/.git/RadeonProRenderUSD/pxr/imaging/plugin/hdRpr/python/generateLightSettingFiles.py", line 38, in <module>
    generate_houdini_ds(args.install, 'Light', light_settings)
  File "/home/redsith/.git/RadeonProRenderUSD/pxr/imaging/plugin/hdRpr/python/houdiniDsGenerator.py", line 48, in generate_houdini_ds
    import hou
  File "/opt/hfs18.5.351/houdini/python2.7libs/hou.py", line 32, in <module>
    import _hou
ImportError: libjemalloc.so.1: cannot open shared object file: No such file or directory
Traceback (most recent call last):
  File "/home/redsith/.git/RadeonProRenderUSD/pxr/imaging/plugin/hdRpr/python/generateFiles.py", line 32, in <module>
    subprocess.check_call([sys.executable, os.path.join(args.scripts_dir, 'generateLightSettingFiles.py'), args.install] + generate_ds_files)
  File "/opt/hfs18.5.351/python/lib/python2.7/subprocess.py", line 190, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/opt/hfs18.5.351/python/bin/python', '/home/redsith/.git/RadeonProRenderUSD/pxr/imaging/plugin/hdRpr/python/generateLightSettingFiles.py', '/home/redsith/.git/RadeonProRenderUSD/build/pxr/imaging/plugin/hdRpr', '--generate_ds_files']' returned non-zero exit status 1
gmake[2]: *** [pxr/imaging/plugin/hdRpr/CMakeFiles/hdRpr.dir/build.make:86: pxr/imaging/plugin/hdRpr/config.h] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:528: pxr/imaging/plugin/hdRpr/CMakeFiles/hdRpr.dir/all] Error 2
gmake: *** [Makefile:160: all] Error 2
```

Thanks

**Comment by hshakula** [comment]
Hm, it's clearly seen that it fails due to the inability to import hou module. It would be possible only if we try to do it from the system python.

Could you please make sure that `sys.executable` on line 32 in `/home/redsith/.git/RadeonProRenderUSD/pxr/imaging/plugin/hdRpr/python/generateFiles.py` points to `/opt/hfs18.5.351/python/bin/python` and not any other python? Just print it

**Comment by Redcroft** [comment]
print(sys.executable) returns
`/opt/hfs18.5.351/python/bin/python`

**Comment by hshakula** [comment]
Hmm, okay, that's what it should be.

And if you run this python, does it fails on `hou` import?

**Comment by Redcroft** [comment]
How strange...

```
[redsith@heimdall build]$ /opt/hfs18.5.351/python/bin/python
Python 2.7.15 (default, Mar 10 2020, 16:24:16) 
[GCC 6.3.1 20170216 (Red Hat 6.3.1-3)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import hou
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named hou
```

**Comment by hshakula** [comment]
Try to run `hython` instead

**Comment by Redcroft** [comment]
hython works

**Comment by hshakula** [comment]
ok, what is the value of `sys.executable` in hython?

**Comment by Redcroft** [comment]
`/opt/hfs18.5.351/python/bin/python`

**Comment by hshakula** [comment]
Well, okay, you can hardcode `hython` as your PYTHON_EXECUTABLE in cmake and `sys.executable` on line 32 in `/home/redsith/.git/RadeonProRenderUSD/pxr/imaging/plugin/hdRpr/python/generateFiles.py`. It will at least generate .ds files for you.

But I will need to set up Fedora to reproduce this issue because there is no such behavior on Ubuntu/CentOS. I don't know when I will have time for this. So if you will figure this out let me know.

**Comment by Redcroft** [comment]
Thanks that worked, but i also notice the same issue under ubuntu 20.04.

Also unless im missing something, where can i control render settings such as trace depth, sampling, etc

**Comment by hshakula** [comment]
So far, I was testing it in Ubuntu 18.04 only.

Check “Render Settings” LOP node

**Comment by Redcroft** [comment]
i tried that node, but the RPR tab only shows Render Quality, and Enable Tone Mapping

**Comment by hshakula** [comment]
Hm, have you built master? Try develop

**Comment by Redcroft** [comment]
Tried using develop branch but the viewport option disappears once selected

**Comment by hshakula** [comment]
It looks like the dynamic library failed to load, set TF_DEBUG=TF_DLOPEN environment variable to get more information about dll loading process

**Comment by hshakula** [comment]
@Redcroft, please try #395. I don't know what g++ you were using to build the plugin but g++ 9.3.0 that is shipped with Ubuntu 20.04 was giving me invalid libraries. #395 fixes it.

**Comment by Redcroft** [comment]
Thanks, that works
